### DISCUSSION TITLE
Changes to version file ignored

### DISCUSSION THREAD

**Discussion by dr4kk4r** [pyinstaller/pyinstaller#7337] [Help]
Changes to version file are ignored by pyinstaller when target python file doesn't change and exe already exists.
This happens when building from file spec or directly from python target file.

The only way for version file to be consumed is to modify target python file which forces pyinstaller to overwrite existing exe.
Is this the desired behavior? Seems to me it should have recognized a change in version file and rebuild the exe.

**Comment by rokm** [comment]
Created issue #7338 from this discussion.
